From ee67d89aa4862508c345a3c4d0c7770ace2382f7 Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Mon, 6 Feb 2023 06:42:34 +0100
Subject: [PATCH] bytesatwork: bawconfig: fix build without EEPROM enabled

Take EEPROM dependency into account for 'builtin2eeprom' command.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
(cherry picked from commit c2b7c6e802efc1a6c6bea626034991003bf5a53e)
---
 board/bytesatwork/baw_config/cmd_baw_config.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/board/bytesatwork/baw_config/cmd_baw_config.c b/board/bytesatwork/baw_config/cmd_baw_config.c
index ded4312eb3..f6e48f566e 100644
--- a/board/bytesatwork/baw_config/cmd_baw_config.c
+++ b/board/bytesatwork/baw_config/cmd_baw_config.c
@@ -146,9 +146,11 @@ struct baw_config_cmd_struct {
 static const struct baw_config_cmd_struct baw_config_cmd[] = {
 #if defined(CONFIG_BAW_CONFIG_BUILTIN)
 	{ "builtin", cmd_builtin },
-	{ "builtin2eeprom", cmd_builtin2eeprom },
 #endif
 #if defined(CONFIG_BAW_CONFIG_EEPROM)
+#if defined(CONFIG_BAW_CONFIG_BUILTIN)
+	{ "builtin2eeprom", cmd_builtin2eeprom },
+#endif
 	{ "read", cmd_read },
 	{ "erase", cmd_erase },
 	{ "write", cmd_write },
-- 
2.39.2

